molecule/upgrade_test/molecule.yml

```diff
@@ -0,0 +1,78 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+lint:
+  name: yamllint
+
+platforms:
+  - name: app-staging
+    box: fpf/securedrop-app
+    box_url: "../vagrant_packager/box_files/app_metadata.json"
+    instance_raw_config_args:
+      - "ssh.insert_key = false"
+    raw_config_args:
+      - "override.vm.synced_folder './', '/vagrant', disabled: true, type: 'nfs'"
+    memory: 1024
+    private_ip: 10.0.1.2
+    groups:
+      - securedrop_application_server
+      - securedrop
+      - staging
+
+  - name: mon-staging
+    box: fpf/securedrop-mon
+    box_url: "../vagrant_packager/box_files/mon_metadata.json"
+    instance_raw_config_args:
+      - "ssh.insert_key = false"
+    raw_config_args:
+      - "override.vm.synced_folder './', '/vagrant', disabled: true, type: 'nfs'"
+    memory: 1024
+    private_ip: 10.0.1.3
+    groups:
+      - securedrop_monitor_server
+      - securedrop
+      - staging
+
+  - name: apt
+    box: bento/ubuntu-14.04
+    instance_raw_config_args:
+      - "vm.synced_folder './', '/vagrant', disabled: true, type: 'rsync'"
+    memory: 256
+    groups:
+      - aptservers
+
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  inventory:
+    links:
+      group_vars: ../../../install_files/ansible-base/group_vars
+      host_vars: ../../../install_files/ansible-base/host_vars
+  env:
+    ANSIBLE_ROLES_PATH: ".molecule/sd-orig/install_files/ansible-base/roles:.molecule/roles"
+  options:
+    e: "@ansible-override-vars.yml"
+  config_options:
+    ssh_connection:
+      pipelining: True
+  playbooks:
+    side_effect: side_effect.yml
+
+scenario:
+  name: upgrade_test
```

Comment:
Nit: can we call this scenario "upgrade"? It'd be more consistent with the naming strategy implicit in Molecule scenarios; e.g. I want to be able to run `molecule test -s upgrade`, rather than `molecule test -s upgrade_test`.